conf/ocsci/acm_2.8.yaml

```diff
@@ -0,0 +1,3 @@
+---
+ENV_DATA:
+  acm_version: "2.8"
```

Comment:
missing new line at the end of file, the same for the other config.

Reply:
done